src/App.tsx

```diff
@@ -15,36 +16,94 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const SORT_FIELD_ALPHABETICCALY = 'alphabeticcaly';
```

Comment:
There is a typo in the constant name `SORT_FIELD_ALPHABETICCALY`. It should be `SORT_FIELD_ALPHABETICALLY` to correctly reflect the intended meaning.